packages/GA4Client/src/commerce-handler.js

```diff
@@ -136,10 +136,147 @@ CommerceHandler.prototype.logCommerceEvent = function (event) {
             null;
     }
 
-    gtag('event', mapGA4EcommerceEventName(event), ga4CommerceEventParameters);
+    this.sendCommerceEventToGA4(
+        mapGA4EcommerceEventName(event),
+        ga4CommerceEventParameters
+    );
     return true;
```

Comment:
I took a look at `mapGA4EcommerceEventName` and it looks like it's possible that `null` may be returned in place of an `eventName`. While it may be an edge case, it's something we should consider handling.

Maybe we can event validate that the response is actually a non-empty string as well.

Probably out of scope for your change but I think it's a good defensive practice so we don't accidentally send an `undefined` event to gtag.

```suggestion
    var eventName = mapGA4EcommerceEventName(event);

    if (typeof eventName === 'string' && eventName.length > 0) {
        this.sendCommerceEventToGA4(
            mapGA4EcommerceEventName(event),
            ga4CommerceEventParameters
        );
    } else {
        return false;
    }
    
    return true;
```